core/src/main/java/org/mule/runtime/core/api/routing/RoutePathNotFoundException.java

```diff
@@ -21,19 +20,19 @@
    */
   private static final long serialVersionUID = -8481434966594513066L;
 
-  public RoutePathNotFoundException(MuleEvent event, MessageProcessor target) {
-    super(event, target);
+  public RoutePathNotFoundException(MessageProcessor target) {
```

Comment:
We need a separate issue to move this exceptions to the API and refactor to extract also the need for the MessageProcessor as property of the exceptions. Can you create one? No need to add a TODO